Synapse/Api/Locker.cs

```diff
@@ -55,22 +55,28 @@ public Vector3 Scale
             }
         }
 
-        public List<LockerChamber> Chambers { get; } = new List<LockerChamber>();
+        public List<LockerChamber> Chambers { get; } = new();
 
+        private LockerType _typeCached = LockerType.Default;
         public LockerType LockerType
         {
             get
             {
-                if (Name.Contains("AdrenalineMedkit")) return Enum.LockerType.MedkitWallCabinet;
-                else if (Name.Contains("RegularMedkit")) return Enum.LockerType.AdrenalineWallCabinet;
-                else if (Name.Contains("Pedestal")) return Enum.LockerType.ScpPedestal;
-                else if (Name.Contains("MiscLocker")) return Enum.LockerType.StandardLocker;
-                else if (Name.Contains("RifleRack")) return Enum.LockerType.RifleRackLocker;
-                else if (Name.Contains("LargeGunLocker")) return Enum.LockerType.LargeGunLocker;
-                return default;
+                if (_typeCached is LockerType.Default) _typeCached = _get();
+                return _typeCached;
+                LockerType _get()
+                {
+                    if (Name.Contains("AdrenalineMedkit")) return LockerType.MedkitWallCabinet;
+                    else if (Name.Contains("RegularMedkit")) return LockerType.AdrenalineWallCabinet;
+                    else if (Name.Contains("Pedestal")) return LockerType.ScpPedestal;
+                    else if (Name.Contains("MiscLocker")) return LockerType.StandardLocker;
+                    else if (Name.Contains("RifleRack")) return LockerType.RifleRackLocker;
+                    else if (Name.Contains("LargeGunLocker")) return LockerType.LargeGunLocker;
+                    return LockerType.Default;
+                };
```

Comment:
@GrafDimenzio 
Question, can the `Locker#Name` ever change once spawned?

Reply:
No the name is just fetched from the gameobject and can't (normally) be changed

Reply:
Well then why the hell do you re-evaluate it every time at runtime 😄

Reply:
I think Warkis did implement this in a PR and the others accepted the change in version 2.8.0 so I never actually inspected the code for the locker object

Reply:
Welp